backend/tests/test_rules_config.py

```
@@ -0,0 +1,449 @@
+    EntryRules,
+    ManagementTriggers,
+    PositionRules,
+    RangeRule,
+    RiskRules,
+    RulesConfig,
+    UniverseRules,
+    load_rules_config,
+)
+
+
+@pytest.fixture()
```

Ruff (0.15.12)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)